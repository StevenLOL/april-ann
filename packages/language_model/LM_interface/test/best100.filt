Darkness had descended like a Curtain by the 
Darkness had descended like a Certain by the 
Darkness had descended Like a Curtain by the 
Derek unless had descended like a Curtain by the 
Darkness had descended Like a Certain by the 
Darkness had descended like a Cecil an by the 
Derek unless had descended like a Certain by the 
Darkness had descended like a Cuban by the 
Darkness had descended like a Curtain by he 
Darkness had descended like a Curtain lay the 
Darkness had descended like a Curtain by The 
Derek was s had descended like a Curtain by the 
Derek mess had descended like a Curtain by the 
Darkness had descended like a Certain by he 
Darkness had descended like a Curtain way the 
Darkness had descended like a Certain lay the 
Darkness had descended like a Certain by The 
Darkness had descended like a Clurman by the 
Derek was s had descended like a Certain by the 
Derek mess had descended like a Certain by the 
Darkness had descended like a Ceuta by the 
Derek Ness had descended like a Curtain by the 
, Darkness had descended like a Curtain by the 
Darkness had descended like a Cecil arm by the 
Derek is less had descended like a Curtain by the 
Derek no less had descended like a Curtain by the 
Dave lawless had descended like a Curtain by the 
Darkness had descended like a Certain way the 
Darkness had descended like a Clerk an by the 
Darkness had descended like a Curtain by He 
Darkness had descended like a Ceuta us by the 
Darkness had descended like a Curtain by she 
Dusk was s had descended like a Curtain by the 
Darkness had descended like a Clerk arm by the 
Darkness had descended like a Court an by the 
Derek Ness had descended like a Certain by the 
, Darkness had descended like a Certain by the 
Derek us less had descended like a Curtain by the 
Dave Lawless had descended like a Curtain by the 
Derek is less had descended like a Certain by the 
Darkness had descended like a Cecil aims by the 
Darkness had descended like a Cecil arms by the 
Darkness had descended like a Cecil aim by the 
Darkness had descended like a Court arm by the 
Derek no less had descended like a Certain by the 
Darkness had descended like a Curtain boy the 
Dave lawless had descended like a Certain by the 
Darkness had descended like a Certain by He 
Darkness had descended like a Curtain , by the 
Darkness had descended like a Certain by she 
Dusk was s had descended like a Certain by the 
Darkness had descended like a Curtain way he 
Plazek unless had descended like a Curtain by the 
Darkness had descended like a Curtain May the 
Darkness had descended like a Curtain Lay the 
Darkness had descended like a Certainly the 
Derek was 5 had descended like a Curtain by the 
Darkness had descended like a Cecil arm lay the 
Derek unless had descended Like a Curtain by the 
Darkness had descended like a Clint an by the 
Darkness had descended like a Clerk aims by the 
Dusky less had descended like a Curtain by the 
Darkness had descended like a Clerk arms by the 
Darkness had descended like a Curtain lay The 
Darkness had descended like a Clerk aim by the 
Darkness had descended like a Cherian by the 
Derek us less had descended like a Certain by the 
Dave Lawless had descended like a Certain by the 
Darkness had descended like a Kaukau by the 
Darkness had descended like a Curan by the 
Darkness had descended like an Curtain by the 
' Darkness had descended like a Curtain by the 
Darkness had descended like a Cecil am by the 
Darkness had descended like a Caetani by the 
Darkness had descended like a Curran by the 
Darkness had descended like a Certain boy the 
Darkness had descended like a Clint arm by the 
Dave caress had descended like a Curtain by the 
Darkness had descended like a , Curtain by the 
Due lawless had descended like a Curtain by the 
Dave kv less had descended like a Curtain by the 
Darkness had descended like a Curtain buy the 
Darkness had descended like a Court aims by the 
Darkness had descended like a Court arms by the 
Darkness , had descended like a Curtain by the 
Darkness had descended like a Clark an by the 
Darkness had descended like a Court aim by the 
Darkness had descended like a Curtain by She 
D merciless had descended like a Curtain by the 
Dark unless had descended like a Curtain by the 
" Darkness had descended like a Curtain by the 
Dusk unless had descended like a Curtain by the 
Darkness had descended like a Clerk arm lay the 
Darkness " had descended like a Curtain by the 
Derek we ss had descended like a Curtain by the 
Darkness had descended Like a Cecil an by the 
Dave kw less had descended like a Curtain by the 
Darkness had descended like a Certain way he 
Plazek unless had descended like a Certain by the 
Darkness had descended like a Certain May the 
