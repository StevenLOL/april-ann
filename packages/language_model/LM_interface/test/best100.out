0||| Darkness had descended like a Curtain by the ||| 2542.038708 -55.259963 8 ||| 1863.131470
0||| Darkness had descended like a Certain by the ||| 2522.371668 -54.448905 8 ||| 1853.013184
0||| Darkness had descended Like a Curtain by the ||| 2579.860477 -62.873960 8 ||| 1811.312134
0||| Derek unless had descended like a Curtain by the ||| 2589.602126 -63.919438 9 ||| 1805.205078
0||| Darkness had descended Like a Certain by the ||| 2560.193315 -62.062901 8 ||| 1801.193726
0||| Darkness had descended like a Cecil an by the ||| 2525.420372 -59.189727 9 ||| 1796.707153
0||| Derek unless had descended like a Certain by the ||| 2569.935207 -63.108379 9 ||| 1795.086914
0||| Darkness had descended like a Cuban by the ||| 2444.409887 -52.965361 8 ||| 1792.517456
0||| Darkness had descended like a Curtain by he ||| 2523.460046 -59.966393 8 ||| 1789.143066
0||| Darkness had descended like a Curtain lay the ||| 2528.246243 -60.403503 8 ||| 1788.783081
0||| Darkness had descended like a Curtain by The ||| 2537.992963 -61.301343 8 ||| 1787.959351
0||| Derek was s had descended like a Curtain by the ||| 2629.158049 -68.591809 10 ||| 1786.212158
0||| Derek mess had descended like a Curtain by the ||| 2587.440190 -65.393468 9 ||| 1785.689087
0||| Darkness had descended like a Certain by he ||| 2503.793005 -59.155335 8 ||| 1779.024780
0||| Darkness had descended like a Curtain way the ||| 2503.952093 -59.187282 8 ||| 1778.807739
0||| Darkness had descended like a Certain lay the ||| 2508.579209 -59.592445 8 ||| 1778.664795
0||| Darkness had descended like a Certain by The ||| 2518.325923 -60.490285 8 ||| 1777.841064
0||| Darkness had descended like a Clurman by the ||| 2462.022572 -55.835806 8 ||| 1776.335815
0||| Derek was s had descended like a Certain by the ||| 2609.491008 -67.780751 10 ||| 1776.093872
0||| Derek mess had descended like a Certain by the ||| 2567.773150 -64.582410 9 ||| 1775.570801
0||| Darkness had descended like a Ceuta by the ||| 2419.803692 -52.420114 8 ||| 1774.330566
0||| Derek Ness had descended like a Curtain by the ||| 2595.544158 -67.142671 9 ||| 1773.199341
0||| , Darkness had descended like a Curtain by the ||| 2499.956234 -59.059953 9 ||| 1772.770874
0||| Darkness had descended like a Cecil arm by the ||| 2559.201078 -64.100634 9 ||| 1772.670776
0||| Derek is less had descended like a Curtain by the ||| 2568.014799 -64.698042 10 ||| 1770.911011
0||| Derek no less had descended like a Curtain by the ||| 2578.292662 -65.690071 10 ||| 1769.509521
0||| Dave lawless had descended like a Curtain by the ||| 2598.734693 -67.795480 9 ||| 1768.704224
0||| Darkness had descended like a Certain way the ||| 2484.285058 -58.376224 8 ||| 1768.689453
0||| Darkness had descended like a Clerk an by the ||| 2536.073096 -62.525065 9 ||| 1768.092285
0||| Darkness had descended like a Curtain by He ||| 2508.969376 -60.530578 8 ||| 1768.010132
0||| Darkness had descended like a Ceuta us by the ||| 2520.344333 -61.303480 9 ||| 1766.745483
0||| Darkness had descended like a Curtain by she ||| 2511.075109 -60.852853 8 ||| 1766.321655
0||| Dusk was s had descended like a Curtain by the ||| 2601.533469 -68.014569 10 ||| 1765.383545
0||| Darkness had descended like a Clerk arm by the ||| 2569.853718 -65.722353 9 ||| 1764.230591
0||| Darkness had descended like a Court an by the ||| 2490.715486 -59.024470 9 ||| 1763.947876
0||| Derek Ness had descended like a Certain by the ||| 2575.877117 -66.331613 9 ||| 1763.081055
0||| , Darkness had descended like a Certain by the ||| 2480.289193 -58.248895 9 ||| 1762.652588
0||| Derek us less had descended like a Curtain by the ||| 2608.032070 -68.877389 10 ||| 1761.723999
0||| Dave Lawless had descended like a Curtain by the ||| 2587.838797 -67.478628 9 ||| 1761.538696
0||| Derek is less had descended like a Certain by the ||| 2548.347759 -63.886984 10 ||| 1760.792725
0||| Darkness had descended like a Cecil aims by the ||| 2540.002381 -63.479502 9 ||| 1760.784790
0||| Darkness had descended like a Cecil arms by the ||| 2541.718528 -63.636343 9 ||| 1760.654419
0||| Darkness had descended like a Cecil aim by the ||| 2557.484783 -65.014680 9 ||| 1760.193237
0||| Darkness had descended like a Court arm by the ||| 2524.496114 -62.221759 9 ||| 1760.086182
0||| Derek no less had descended like a Certain by the ||| 2558.625744 -64.879012 10 ||| 1759.391357
0||| Darkness had descended like a Curtain boy the ||| 2506.757421 -61.089903 8 ||| 1759.213135
0||| Dave lawless had descended like a Certain by the ||| 2579.067653 -66.984422 9 ||| 1758.585938
0||| Darkness had descended like a Certain by He ||| 2489.302335 -59.719520 8 ||| 1757.891846
0||| Darkness had descended like a Curtain , by the ||| 2473.934442 -58.155214 9 ||| 1757.400757
0||| Darkness had descended like a Certain by she ||| 2491.408069 -60.041795 8 ||| 1756.203369
0||| Dusk was s had descended like a Certain by the ||| 2581.866429 -67.203511 10 ||| 1755.265259
0||| Darkness had descended like a Curtain way he ||| 2485.373377 -59.652632 8 ||| 1754.750366
0||| Plazek unless had descended like a Curtain by the ||| 2567.448472 -66.337487 9 ||| 1754.583252
0||| Darkness had descended like a Curtain May the ||| 2510.328266 -61.794653 8 ||| 1754.486816
0||| Darkness had descended like a Curtain Lay the ||| 2530.756183 -63.545803 8 ||| 1754.298096
0||| Darkness had descended like a Certainly the ||| 2346.706615 -48.221954 7 ||| 1754.199341
0||| Derek was 5 had descended like a Curtain by the ||| 2574.826421 -66.726034 10 ||| 1753.846680
0||| Darkness had descended like a Cecil arm lay the ||| 2545.408624 -64.561432 9 ||| 1753.453247
0||| Derek unless had descended Like a Curtain by the ||| 2627.423895 -71.533435 9 ||| 1753.385742
0||| Darkness had descended like a Clint an by the ||| 2487.809589 -59.716877 9 ||| 1752.890137
0||| Darkness had descended like a Clerk aims by the ||| 2550.655022 -65.101221 9 ||| 1752.344604
0||| Dusky less had descended like a Curtain by the ||| 2535.334539 -63.808474 9 ||| 1752.243896
0||| Darkness had descended like a Clerk arms by the ||| 2552.371169 -65.258062 9 ||| 1752.214233
0||| Darkness had descended like a Curtain lay The ||| 2524.200425 -63.174396 8 ||| 1752.114990
0||| Darkness had descended like a Clerk aim by the ||| 2568.137423 -66.636399 9 ||| 1751.753052
0||| Darkness had descended like a Cherian by the ||| 2440.055714 -56.062923 8 ||| 1751.695068
0||| Derek us less had descended like a Certain by the ||| 2588.365030 -68.066330 10 ||| 1751.605713
0||| Dave Lawless had descended like a Certain by the ||| 2568.171757 -66.667569 9 ||| 1751.420410
0||| Darkness had descended like a Kaukau by the ||| 2439.553499 -56.060848 8 ||| 1751.217285
0||| Darkness had descended like a Curan by the ||| 2436.740779 -55.824822 8 ||| 1751.183350
0||| Darkness had descended like an Curtain by the ||| 2455.723265 -57.461877 8 ||| 1750.892456
0||| ' Darkness had descended like a Curtain by the ||| 2502.452205 -61.171019 9 ||| 1750.412842
0||| Darkness had descended like a Cecil am by the ||| 2523.396583 -63.005261 9 ||| 1749.762329
0||| Darkness had descended like a Caetani by the ||| 2435.255706 -55.833752 8 ||| 1749.593140
0||| Darkness had descended like a Curran by the ||| 2437.663397 -56.067746 8 ||| 1749.245972
0||| Darkness had descended like a Certain boy the ||| 2487.090375 -60.278844 8 ||| 1749.094849
0||| Darkness had descended like a Clint arm by the ||| 2521.590206 -62.914165 9 ||| 1749.028442
0||| Dave caress had descended like a Curtain by the ||| 2575.610859 -67.519021 9 ||| 1748.835205
0||| Darkness had descended like a , Curtain by the ||| 2466.519179 -58.257024 9 ||| 1748.786865
0||| Due lawless had descended like a Curtain by the ||| 2601.550350 -69.745792 9 ||| 1748.558472
0||| Dave kv less had descended like a Curtain by the ||| 2675.093019 -75.698365 10 ||| 1748.480225
0||| Darkness had descended like a Curtain buy the ||| 2491.020488 -60.674863 8 ||| 1748.362549
0||| Darkness had descended like a Court aims by the ||| 2505.297418 -61.600627 9 ||| 1748.200195
0||| Darkness had descended like a Court arms by the ||| 2507.013565 -61.757468 9 ||| 1748.069824
0||| Darkness , had descended like a Curtain by the ||| 2471.174886 -58.741124 9 ||| 1747.743164
0||| Darkness had descended like a Clark an by the ||| 2497.395788 -60.979677 9 ||| 1747.609131
0||| Darkness had descended like a Court aim by the ||| 2522.779941 -63.135805 9 ||| 1747.608765
0||| Darkness had descended like a Curtain by She ||| 2497.990426 -61.334628 8 ||| 1747.564941
0||| D merciless had descended like a Curtain by the ||| 2568.628330 -67.052969 9 ||| 1747.339600
0||| Dark unless had descended like a Curtain by the ||| 2540.120352 -64.633467 9 ||| 1747.316895
0||| " Darkness had descended like a Curtain by the ||| 2493.773533 -60.758751 9 ||| 1746.587891
0||| Dusk unless had descended like a Curtain by the ||| 2561.977595 -66.664588 9 ||| 1745.261353
0||| Darkness had descended like a Clerk arm lay the ||| 2556.061265 -66.183152 9 ||| 1745.013062
0||| Darkness " had descended like a Curtain by the ||| 2460.004415 -58.029488 9 ||| 1744.950928
0||| Derek we ss had descended like a Curtain by the ||| 2666.266739 -75.252096 10 ||| 1744.907959
0||| Darkness had descended Like a Cecil an by the ||| 2563.242141 -66.803724 9 ||| 1744.887817
0||| Dave kw less had descended like a Curtain by the ||| 2673.899346 -75.909524 10 ||| 1744.800537
0||| Darkness had descended like a Certain way he ||| 2465.706343 -58.841574 8 ||| 1744.632080
0||| Plazek unless had descended like a Certain by the ||| 2547.781554 -65.526428 9 ||| 1744.465088
0||| Darkness had descended like a Certain May the ||| 2490.661220 -60.983594 8 ||| 1744.368530
